engine/crates/composition/Cargo.toml

```diff
@@ -2,9 +2,13 @@
 name = "grafbase-composition"
 version = "0.1.0"
 edition = "2021"
+license = "Apache-2.0"
+description = "An implementation of Grafbase federated schema composition"
+repository = "https://github.com/grafbase/grafbase"
```

Comment:
should this be https://github.com/grafbase/grafbase/tree/main/engine/crates/composition ?

Reply:
done